yy_parser.go

```diff
@@ -158,7 +158,24 @@ func (parser *Parser) Parse(sql, charset, collation string) (stmt []ast.StmtNode
 		warns = nil
 	}
 	if len(errs) != 0 {
-		return nil, warns, errors.Trace(errs[0])
+		err := errs[0]
```

Comment:
It changes the logistic, can you please explain why do not return the error in some situations?

Reply:
Maybe it's not the best solution, I'll close it.